Corona-Warn-App/src/main/java/de/rki/coronawarnapp/environment/EnvironmentSetup.kt

```diff
@@ -115,6 +117,8 @@ class EnvironmentSetup @Inject constructor(
         get() = getEnvironmentValue(DOWNLOAD).asString
     val dataDonationCdnUrl: String
         get() = getEnvironmentValue(DATA_DONATION).asString
+    val traceLocationCdnUrl: String
```

Comment:
Please also display the new endpoint in the debug menu for the testers.

Reply:
Done